### ISSUE TITLE
fbneo and pgen don't work without vulkan

### ISSUE THREAD

**Issue by carlosgamer23** [Cpasjuste/pemu#262]
The previous version did not require vulkan, the current one says that a vulkan dll is missing (I do not have vulkan on my video card, and I believe that many use this emulator because it does not require it, having or not vulkan should be optional)

**Comment by Cpasjuste** [comment]
Hi,

I guess this is on windows platform ? Here I do not have this problem on my windows 10 development platform (with Nvidia). 
Can you confirm the problem is on windows and provide me your graphic card model ?

**Comment by carlosgamer23** [comment]
> Hi,
> 
> I guess this is on windows platform ? Here I do not have this problem on my windows 10 development platform (with Nvidia). Can you confirm the problem is on windows and provide me your graphic card model ?
> 
> If you can paste the output of the console it could help too.
> 
> I guess the problem is related to the SDL2 package.

Yes, it's Windows, I use an AMD Radeon: AMD Radeon HD 5450 (It's understandable for someone who uses NVIDIA to forget that AMD exists haha, but thanks for answering me)

**Comment by carlosgamer23** [comment]
![erroo do pgen](https://github.com/Cpasjuste/pemu/assets/144261319/e677cfb8-db88-476c-9440-2bef6b1f5a32)

**Comment by Cpasjuste** [comment]
I guess this is the same problem as seen here: https://github.com/ppy/osu-framework/issues/5702
I guess this file/dll comes with the Nvidia driver and not the AMD one (maybe...). I will try to upload this dll here to see if it's enough for you, so I can just add it to the next release.

**Comment by Cpasjuste** [comment]
Here it is, if you have the time to test it (copy the file to the emulator directory).
[vulkan-1.zip](https://github.com/Cpasjuste/pemu/files/12616551/vulkan-1.zip)

**Comment by MayanKoyote** [comment]
Vulkan is distributed with AMD graphics card drivers.
https://www.amd.com/en/support/kb/release-notes/rn-rad-win-vulkan
Even for vintage video cards starting from CGN 1st gen (HD 77X0, 2012-2013, and up).
https://www.amd.com/en/technologies/vulkan
But not for prehistoric ones (HD 5450, 2010).

**Comment by Cpasjuste** [comment]
I see, thanks for the clarification. 

I just submitted a "fix" which copy the dll to the installation folder, but after thinking at it I think it's a bad way of fixing that.
I think I should be able to disable the Vulkan sdl2 video driver at runtime to prevent this (unless something is wrong with sdl2). Latest sdl2 probably want to use Vulkan by default.

**Comment by carlosgamer23** [comment]
This Vulkan dll worked perfectly on emulators (they are opening correctly here), but this version of pgen is not recognizing the CHD files that I have, it is correctly reading the CUE/BIN games that are in the same folder as my CHD games , however, did not identify the prototype of Sonic CD and Shining Force (which are in the CHD format), along with the CUE/BIN games that are in the same folder.

 I tested two keyboards on the emulator, and I saw that even switching between them, none of them can make the space key (44) interact like the A button, to select the game it is necessary to use the key corresponding to the A button, but as do this on Windows if the emulator ignores it?  I use a joystick to save the emulator settings, because I can't do this using any keyboard I use (even changing the select and start keys on the emulator, I still can't close it using the keyboard, because even opening the MENU and going to "quit", no key that I set to work as the "A" confirmation key, closes the emulator, the space key works as the "A" key only within the game and not in the game's initial menu, so I have to use a joystick to open and confirm (with A) that I will enter the game, as well as use the same to close the emulator)

I believe it would be positive (if possible) to have the option to choose where the rom folders are located directly in the emulator, like those in FBA and MAME, instead of having to manually edit the config, but congratulations on your work!  It's the best project we have to contemplate!

English isn't really my strong point, but if you ever want to translate the emulator into other languages, you can count on me.  I will make a point of helping this project!
